test/e2e/page-objects/pages/confirmations/redesign/transaction-confirmation.ts

```diff
@@ -186,27 +202,88 @@ class TransactionConfirmation extends Confirmation {
     await this.driver.clickElement(this.customNonceButton);
   }
 
-  async clickCustomNonceSaveButton() {
-    await this.driver.clickElement(this.customNonceSaveButton);
-  }
-
   async clickGasFeeTokenPill() {
     await this.driver.clickElement(this.gasFeeTokenArrow);
   }
 
+  async clickSaveButton() {
+    await this.driver.clickElement(this.saveButton);
+  }
+
   async closeGasFeeToastMessage() {
     // the toast message automatically disappears after some seconds, so we need to use clickElementSafe to prevent race conditions
     await this.driver.clickElementSafe(this.gasFeeCloseToastMessage, 5000);
   }
 
+  /**
+   * Edits the gas fee by setting custom gas limit and price values
+   *
+   * @param gasLimit - The gas limit value to set
+   * @param gasPrice - The gas price value to set
+   */
+  async editGasFeeLegacy(gasLimit: string, gasPrice: string): Promise<void> {
```

Comment:
this method was on the token class page, but it's now moved here, as it can be used by any transaction (not only token transfers)